contrib/coreos/user-data.example

```diff
@@ -176,3 +190,70 @@ write_files:
     content: |
       [Coredump]
       Storage=none
+  - path: /etc/systemd/system/ntpd.service.d/debug.conf
```

Comment:
This unit and the following fleet custom binary should be removed - botched rebase, it looks like.